docs/src/test/java/docs/http/javadsl/server/JsonStreamingExamplesTest.java

```diff
@@ -49,10 +49,6 @@
 
   //#routes
   final Route tweets() {
-    //#formats
-    final Unmarshaller<ByteString, JavaTweet> JavaTweets = Jackson.byteStringUnmarshaller(JavaTweet.class);
-    //#formats
```

Comment:
In Java, this Unmarshaller is not used at the beginning of the "JSON Streaming" section. So removing it.